app/javascript/components/search/controls/CombinedFacetControl.js

```diff
@@ -0,0 +1,123 @@
+import React, { useContext, useState } from 'react'
+import _find from 'lodash/find'
+
+import { StudySearchContext } from 'providers/StudySearchProvider'
+import { SearchFacetContext } from 'providers/SearchFacetProvider'
+import FiltersBoxSearchable from './FiltersBoxSearchable'
+import { ApplyButton, ClearFilters } from './FiltersBox'
+import { SearchSelectionContext, isFacetApplicable, getSelectionForFacet } from 'providers/SearchSelectionProvider'
+import useCloseableModal from 'hooks/closeableModal'
+import _kebabCase from 'lodash/kebabCase'
+import _compact from 'lodash/compact'
+import _flatten from 'lodash/flatten'
+import _clone from 'lodash/clone'
+
+/**
+ * Component for combining two facets into a single panel control.
+ * They will be rendered more or less independently, but with a single 'apply' button
+ *
+ * Note that this does NOT support numeric facets -- to support those the display CSS would have to be
```

Comment:
This line is 101 characters long, which I believe throws a linting error.

If so, to keep linting high-signal, please either shorten long lines like this (preferable) or adjust the line length (`max-len`) style rule to something more lenient.  I prefer 80 characters, but not strongly.

Current style rule:
```
./node_modules/.bin/eslint --print-config app/javascript/components/search/controls/KeywordSearch.js  | grep -A 8 max-len
    "max-len": [
      2,
      {
        "code": 80,
        "tabWidth": 2,
        "ignoreUrls": true,
        "ignorePattern": "goog.(module|require)"
      }
    ],
```

Reply:
yeah, I remember the discussion when we agreed on 80 -- our code is unfortunately chock-full of longer lines (I'm definitely a guilty party, though not the only one).  Let's have a brief chat with Eno tomorrow about whether it's worth cleaning them all up, or increasing the length.  (I lean towards longer lines, but not strongly, so she can break the tie :)   Once we settle that, then we can add linting to the server-side PR checks

Reply:
Sounds good!